src/Utilities/SingleStackUtils/SingleStackUtils.jl

```diff
@@ -0,0 +1,113 @@
+module SingleStackUtils
+
+export get_vars_from_nodal_stack, get_vars_from_element_stack
+
+using OrderedCollections
+using StaticArrays
+
+using ClimateMachine.DGmethods
```

Comment:
```suggestion
using ...DGmethods
```
Shouldn't we use relative paths here?

Reply:
Let's fix it when we squash.